[Feature] Projections - config + create projection with trackEmittedStreams (#92)
RemoteMetering/geteventstore-promise: README.md

# Supported Methods

* start(projectionName)
    #### projectionName
    The name of the projection
* stop(projectionName)
    #### projectionName
    The name of the projection
* reset(projectionName)
    #### projectionName
    The name of the projection
* remove(projectionName)
    #### projectionName
    The name of the projection
* config(projectionName)
    #### projectionName
    The name of the projection
* getState(projectionName, options)
    #### projectionName
    The name of the projection

    #### options

>     Object used to specify the partition to query the state for eg. `{ partition: 1}`

Reviewer: ```suggestion
    Object, `partition` used to specify the partition to query the state with. e.g. `{ partition: 1 }`
```

Author: Thanks and sorry for my poor English :)

    #### emitEnabled(optional)
    Should enable emitting, defaults to false

    #### trackEmittedStreams(optional)

>     Should track the emitted streams (used when we want delete the emitted streams when we delete the projection), defaults to false

Reviewer: ```suggestion
    Should track the emitted streams (tracking emitted streams enables you to delete a projection and all the streams that it has created), defaults to false
```
